libs/ngx-charts-on-fhir/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@elimuinformatics/ngx-charts-on-fhir",
-  "version": "7.0.1",
+  "version": "7.0.2",
```

Comment:
This is a breaking change because it's no longer compatible with Angular 16-18. Bump the major version for a breaking change.